fallback.env.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
See comment in jest.config.js regarding this file - if you simplify the logic in there, you won't need this file at all.

Reply:
makes sense, yes.